projects/admin/src/app/record/custom-editor/circulation-settings/circulation-policy-form.service.ts

```diff
@@ -29,7 +30,8 @@ export class CirculationPolicyFormService {
 
   constructor(
     private apiService: ApiService,
```

Comment:
Remove this service (unused)

Reply:
Actually still useful as jma asked to use a function from this service.